services/integrationServices.js

```diff
@@ -0,0 +1,160 @@
+            if (integrationSettings.loginNotify === true) {
+                console.log("Slack Login Event enabled");
+                text = "A User just Logged In"
+                title = "Login";
+                color = "#36a64f"
+            } else {
+                console.log("Slack Login Event Disabled");
+            }
+            break;
+        case "Signup":
+            if (integrationSettings.signUpNotify === true) {
+                console.log("Slack Sign Up Event enabled");
+                text = "A New User just Signed Up";
+                title = "Sign Up";
+                color = "#5CACEE";
+            } else {
+                console.log("Slack SingUp Event Disabled");
+            }
+            break;
+        default:
+            console.log(event_type + " Event enabled");
+            title = event_type;
+            color = "#9932CC";
+            text = "A User just created an Event";
+    }
+    if (text) {
+        slack.webhook({
+            channel: "#test",
+            username: "CloudBoost",
+            attachments: [
+                {
+                    "fallback": "Whenever a User Signs Up a notification will appear here",
+                    "color": color,
+                    "pretext": "Optional text that appears above the attachment block",
+                    "author_name": "Event Notifications",
+                    "author_link": "https://www.cloudboost.io/",
+                    "author_icon": "https://d1qb2nb5cznatu.cloudfront.net/startups/i/490103-917cc2864d0246e313e9521971422f09-medium_jpg.jpg?buster=1430997518",
+                    "title": "User " + event_type + " Notification",
+                    "title_link": "https://www.cloudboost.io/",
+                    "text": text,
+                    "fields": [
+                        {
+                            "title": "User Name",
+                            "value": user,
+                            "short": false
+                        },
+                        {
+                            "title": "User Email",
+                            "value": user_email,
+                            "short": false
+                        }
+                    ],
+                    "footer": "CloudBoost",
+                    "footer_icon": "https://d1qb2nb5cznatu.cloudfront.net/startups/i/490103-917cc2864d0246e313e9521971422f09-medium_jpg.jpg?buster=1430997518",
+                    "ts": timeStamp
+                }
+            ],
+            icon_emoji: "https://d1qb2nb5cznatu.cloudfront.net/startups/i/490103-917cc2864d0246e313e9521971422f09-medium_jpg.jpg?buster=1430997518"
+        }, function (err, response) {
+            if (!err) {
+                console.log(response);
+            } else {
+                console.log(err);
+            }
+        });
+        return true;
+    }
+    return false;
+}
+
+function notifyOnZapier(integrationSettings, event_type, user, user_email) {
+    var text;
```

Comment:
Complete Zapier Integration. Fix text like in Slack, and fix this function like in Slack.